source/style-guide/style/text-formatting.txt

```diff
@@ -4,603 +4,1636 @@
+   Terms, new
+     Format using *italics*.
 
-       **Exception:** Show window, dialog box, wizard, page, panel, and
-       screen names in regular text unless they need to be
-       distinguished from the surrounding text. In such cases, use
-       bold.
-     - In the Microsoft Exchange dialog box, click **Apply** and then
-       click **OK**.
+     .. list-table::
+        :widths: 50 50
+        :header-rows: 1
 
-       On the Choose Service page, select **Microsoft Exchange or
-       compatible service**, and then click **Next**.
+        * - Code Example
+          - Rendered Example
 
-       Read the preliminary steps in the Configure Your Server wizard,
-       and then click **Next**.
-
-   * - URLs (not live)
-     - Bold
-     - To access the web interface, open your web browser and navigate
-       to **http://yourDomain.com/zipit-install.php**.
-
-   * - URLs (live)
-     - See :ref:`links-and-cross-references`.
-     - Not applicable
-
-   * - User input
-     - *(GUI)* Bold
-
-       *(CLI)* Monospace
-     - *(GUI)* In the **Server** text box, type **outlook**.
-
-       *(CLI)* Create a file by using a text editor, and insert the
-       following code: ``<?php phpinfo(); ?>``
-
-       *(CLI)* For the username, enter ``admin``.
-
-   * - Variable (placeholder) text
-     - See :ref:`placeholder-variable-text`.
-     - Not applicable
-
-   * - Variables, environment
-     - Monospace
-     - You can set the ``MYSQL_HOST`` environment variable to the
-       remote host's address.
-
-       You can export the tenant ID to the ``$account`` environment
-       variable and the authentication token to the ``$token``
-       environment variable.
-
-   * - Window names
-     - Regular text
-     - In the Network Connections window, right-click the private
-       adapter and select **Properties**.
-
-   * - Wizard names and wizard page names
-     - Regular text
-     - On the Welcome page of the Active Directory Domain Services
-       Installation Wizard, ensure that the **Use advanced mode
-       installation** check box is cleared, and then click **Next**.
-
-   * - Words as words
-     - Italic
-     - Don't use *button* and *icon* interchangeably. If you're
-       referring to a command button or toolbar button (labeled or
-       unlabeled), use *button*. If you're referring to a graphic on a
-       screen, window, or other area, use *icon*.
+        * - .. code-block:: rst
+
+               MongoDB instances that are built using the base Linux
+               images are created without a dedicated swap partition
+               and with *swappiness* (a measure of how the Linux kernel
+               tries to use swap memory) set to 0.
+
+          - MongoDB instances that are built using the base Linux
+            images are created without a dedicated swap partition and
+            with *swappiness* (a measure of how the Linux kernel tries
+            to use swap memory) set to 0.
+
+   Terms, unique sense
```

Comment:
Could you explain "unique sense"? Are these "scare quotes"? If so, we should probably discourage usage completely.